scripts/proto_to_js.js

```diff
@@ -0,0 +1,15 @@
+var lines = [];
+var lineReader = require('line-reader');
+
+lineReader.eachLine('node_modules/grpc-bus/proto/grpc-bus.proto', function(line, last) {
```

Comment:
Why? Because it relies on a library to get each line? Seems like just syntactic sugar to me. If you're worried we can do it w lower level libraries (just `fs`).